faucet/models.py

```diff
@@ -5,38 +5,53 @@
 from django.db import models
 
 # Create your models here.
-from django.db.models import Count, Sum
+from django.db.models import Sum
 from django.utils import timezone
+from constance import config
 
 
 class FaucetTransaction(models.Model):
 
-    TOKEN_PER_DAY_MAXIMUM = 200
-
-    public_key = models.CharField(max_length=128, primary_key=True)
+    public_key = models.CharField(max_length=128, db_index=True)
     amount = models.FloatField()
```

Comment:
A float field for money! :scream:
Please use an Integer or Decimal Field instead